### DISCUSSION TITLE
为什么绳网的显示界面没有文字或图片

### DISCUSSION THREAD

**Discussion by amns66615** [share121/inter-knot#2967] [其他]
我断定他所说的绳网小助手应该就是绳网跨域助手，我昨天刚把那个脚本弄上。可刚进去的时候啥都没有。谁能给我解释一下？挺急的。

**Comment by csy20070709** [comment]
也许你需要更好的魔法？

**Comment by Ziew-Ryan** [comment]
Microsoft Store，下个Watt Toolkit(曾经的Steam++)试试

**Comment by Ziew-Ryan** [comment]
![image](https://github.com/user-attachments/assets/176e0102-bbfe-473a-8818-21a550224822)
就是这个

**Comment by amns66615** [comment]
谢谢



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月19日(周一) 上午10:47
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [share121/inter-knot] 为什么绳网的显示界面没有文字或图片 (Discussion #2967)




 
image.png (view on web)
 就是这个

**Reply by Ziew-Ryan** [reply]
不用谢

**Comment by amns66615** [comment]
对了，那个网页我是找到了。是下载正常的那个还是下载游戏加速器？



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月19日(周一) 上午10:57
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [share121/inter-knot] 为什么绳网的显示界面没有文字或图片 (Discussion #2967)




 
不用谢

**Reply by Ziew-Ryan** [reply]
https://steampp.net/进去这个网址（官网），选择Windows下载

**Reply by Ziew-Ryan** [reply]
进这个：htps://steampp.net/
